packages/mdc-list/component.ts

```diff
@@ -237,10 +238,7 @@ export class MDCList extends MDCComponent<MDCListFoundation> {
   private handleKeydownEvent_(evt: KeyboardEvent) {
     const index = this.getListItemIndex_(evt);
     const target = evt.target as Element;
-
-    if (index >= 0) {
```

Comment:
why do you not need the check anymore?

Reply:
Now that list handles the keyboard events on the root, we no longer need to limit the keyboard handle to list items.